architecture/additional_concepts/storage.adoc

```diff
@@ -68,26 +68,26 @@ administrator configures one or more dynamic provisioners that provision storage
 and a matching PV.
 
 Alternatively, a cluster administrator can create a number of PVs in advance,
-which carry the details of the real storage that is available for use by cluster
-users. PVs exist in the API and are available for consumption.
+which carry the details of the real storage that is available for use. PVs exist
+in the API and are available for consumption.
 
 [[binding]]
 
 === Binding
 
-A user creates a `PersistentVolumeClaim` with a specific amount of storage
-requested and with certain access modes and optionally a `StorageClass`. A
-control loop in the master watches for new PVCs. It either finds a matching PV
-or waits for a provisioner for the `StorageClass` to create one, then binds them
-together.
+Create a PVC with a specific amount of storage requested and with certain access
```

Comment:
Maybe 
When you create a PVC, you request a specific amount of storage, specify the required access mode, and can create a StorageClass to describe and classify the storage. The control loop in the master watches for new PVCs and binds the new PVC to an appropriate PV. If an appropriate PV does not exist, a provisioner for the StorageClass creates one.